Classes/TBODeveloperOverlayKVDebugger/TBODeveloperOverlayKVDebugger.m

```diff
@@ -96,4 +106,17 @@ - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInte
     return [self.datasource titleForSection:section];
 }
 
+#pragma mark helper
+
+- (Class)detailViewControllerClassForValue:(id)value {
```

Comment:
I guess it would be better to ask for a viewControllerClassForValueClass instead of the value itself.